maquetado/index.html

```diff
@@ -9,14 +9,71 @@
 </head>
 <body>
   <!-- NAV -->
-  <!-- TODO -->
+  <nav class="nav">
+    <img alt="Wolox" class="nav-logo" src="assets/wolox_logo.svg">
```

Comment:
Use an `alt` attribute please

Reply:
The alt is present... Does it has to be at the end?